microscopium/screens/cellomics.py

```diff
@@ -132,12 +137,12 @@ def stack_channels(images, order=[0, 1, 2]):
     dtype = images[0].dtype
     concat_image = np.zeros((m, n, 3), dtype=dtype)
```

Comment:
Actually, this is a lot easier than I realised: reorder the images using the `channel_order` argument, then use `np.dstack`.